docs/tutorial/dynamic-introspection.md

```diff
@@ -75,7 +75,7 @@ input RecipeInput {
 
 ## How can we test it?
 
-In order to simulate the non introspectable feature. Be sure that your application is running.
+To simulate the non-introspectable feature, make sure that your application is running.
```

Comment:
`In order to` -> `make sure` ?

Reply:
`In order to` can be replaced by `To` alone, but in any case, the sentence is too short and doesn't work. Also, it's usually better to write `make sure blablabla` that `be sure blablabla`.